arithmetization/src/main/java/net/consensys/linea/plugins/config/LineaTracerSharedCliOptions.java

```diff
@@ -33,6 +36,14 @@ public class LineaTracerSharedCliOptions implements LineaCliOptions {
           "If the sequencer needs to use or not the limitless prover (default: ${DEFAULT-VALUE})")
   private boolean limitlessEnabled = DEFAULT_LIMITLESS_ENABLED;
 
+  @CommandLine.Option(
+      names = {COUNT_HISTORICAL_BLOCKHASHES_ENABLED},
+      hidden = true,
+      paramLabel = "<BOOLEAN>",
+      description =
+          "If the sequencer/coordinator needs to count the lines from historical blockhashes (default: ${DEFAULT-VALUE})")
+  private boolean countHistoricalBlockHashes = DEFAULT_COUNT_HISTORICAL_BLOCKHASHES_ENABLED;
```

Comment:
Sanity check: `countHistoricalBlockHashes` is given the default value ? Rather than `COUNT_HISTORICAL_BLOCKHASHES_ENABLED` ?

**Edit:** I see it's set to the real value in `fromConfig`